src/gluonts/distribution/distribution_output.py

```diff
@@ -82,12 +82,22 @@ class Output:
     """
 
     args_dim: Dict[str, int]
+    _dtype: DType = np.float32
+
+    @property
+    def dtype(self):
+        return self._dtype
+
+    @dtype.setter
+    def dtype(self, dtype: DType):
+        self._dtype = dtype
```

Comment:
I’m wondering why using this getter/setter pair here? Anywhere else this is just stored as an attribute

Reply:
This is for setting the `dtype` of the distribution output correctly in case the default used [here](https://github.com/awslabs/gluon-ts/blob/e021fa4e20cb562f836211f716602b359c79af36/src/gluonts/model/deepar/_estimator.py#L133) is not consistent with the  `dtype` passed to the estimator.

Reply:
I'm not sure I understand. This looks like you just use `dtype` as an alias for `_dtype`.

Reply:
just defining `dtype` as a property to avoid ugly getter and setter methods and also not to access private fields directly from outside.